packages/firebase_analytics/android/src/main/java/io/flutter/plugins/firebaseanalytics/FirebaseAnalyticsPlugin.java

```diff
@@ -16,20 +22,55 @@
 import java.util.Map;
 
 /** Flutter plugin for Firebase Analytics. */
-public class FirebaseAnalyticsPlugin implements MethodCallHandler {
-  private final PluginRegistry.Registrar registrar;
-  private final FirebaseAnalytics firebaseAnalytics;
+public class FirebaseAnalyticsPlugin implements MethodCallHandler, FlutterPlugin, ActivityAware {
```

Comment:
Have you run the new implementation manually to verify that it works as expected?

Reply:
Yes I have run it and it work as expected, the example app that is.